AndroidSDKCore/src/main/java/com/leanplum/Leanplum.java

```diff
@@ -900,8 +900,10 @@ private static void handleStartResponse(final JSONObject response) {
           }
         }
 
-        String token = response.optString(Constants.Keys.TOKEN, null);
-        APIConfig.getInstance().setToken(token);
+        String token = response.optString(Constants.Keys.TOKEN);
```

Comment:
optString returns empty string as default. Passing ```null``` as default generates a lint warning because the parameter is marked as ```@NonNull```